src/utils/price.ts

```diff
@@ -42,7 +45,10 @@ export function getEthPriceInUSD(): BigDecimal {
   );
 
   if (
-    daiPool.token0().toHexString().toLowerCase() == WETH_ADDRESS.toLowerCase()
+    daiPool
```

Comment:
Could this stuff be cased on the chain ID? Seems like we need it.

Reply:
created #47 to track this. will include as part of upgrading the repo to support multiple chains